plugins/quickinspector/geometryextension/sggeometryextension.cpp

```diff
@@ -36,12 +36,13 @@
 using namespace GammaRay;
 
 SGGeometryExtension::SGGeometryExtension(PropertyController *controller)
-    : SGGeometryExtensionInterface(controller->objectBaseName() + ".sgGeometry", controller)
-    , PropertyControllerExtension(controller->objectBaseName() + ".sgGeometry")
+    : PropertyControllerExtension(controller->objectBaseName() + ".sgGeometry")
     , m_node(0)
-    , m_model(new SGGeometryModel(this))
+    , m_vertexModel(new SGVertexModel())
+    , m_adjacencyModel(new SGAdjacencyModel())
```

Comment:
While correct from a memory management POV, this will "leak" the two models into the user-visible object tree. If you give them a GammaRay object as parent, they will be filtered out.